default.nix

```diff
@@ -41,14 +41,27 @@ in
 # parameterize this file when targetting different
 # hosts.
 { system ? builtins.currentSystem
+, crossSystem ? null
 , config ? {}
-, pkgs ? commonLib.getPkgs { inherit config system; }
-, ... }@args:
-# We will instantiate the default-nix template with the
-# nix/pkgs.nix file...
-commonLib.nix-tools.default-nix ./nix/pkgs.nix args
-# ... and add additional non-haskell packages we want to build on CI:
-// {
+, pkgs ? commonLib.getPkgs { inherit config crossSystem system; }
+, withHoogle ? true
+}:
+let 
+  # We will instantiate the default-nix template with the
+  # nix/pkgs.nix file...
+  defaultNix = commonLib.nix-tools.default-nix ./nix/pkgs.nix { 
+    inherit system crossSystem config pkgs;
+  };
+in defaultNix // {
+  # ... and add additional packages we want to build on CI:
+
+  env = defaultNix.nix-tools.shellFor {
+    inherit withHoogle;
+    # env will provide the dependencies of cardano-shell
+    packages = ps: with ps; [ cardano-shell ];
```

Comment:
Do we have the stack coveralls package dependendency somewhere in the default package set? If not, it would be useful, since we could all have the same version and run the same thing.

Reply:
We do (defined in `iohk-nix`). I added it to the default nix-shell.